crates/storage/src/collections/lazy_map.rs

```diff
@@ -408,6 +408,25 @@ where
         Ok(previous)
     }
 
+    /// Same as `LazyMap::insert` above but does not add the key-value to the
```

Comment:
this is not needed anywhere atm right? I'd say let's skip it as I think we'd also need another version of `fn remove`. For lazy collections though I think we can do better, we could parametrize the handlers to allow to specify the `WriteOpts` at type level and apply it to all its sub-keys

Reply:
https://github.com/anoma/namada/pull/2355/commits/eea0a55f079fbb9844446e009553214cdc6f2c98